src/commands/run.ts

```diff
@@ -176,45 +177,26 @@ export class RunCommand extends ApifyCommand<typeof RunCommand> {
 		let CRAWLEE_PURGE_ON_START = '0';
 
 		// Purge stores
-		// TODO: this needs to be cleaned up heavily - ideally logic should be in the project analyzers
 		if (this.flags.purge) {
-			switch (projectType) {
-				case PROJECT_TYPES.PRE_CRAWLEE_APIFY_SDK: {
-					await Promise.all([purgeDefaultQueue(), purgeDefaultKeyValueStore(), purgeDefaultDataset()]);
-					info({ message: 'All default local stores were purged.' });
-					break;
-				}
-				case PROJECT_TYPES.CRAWLEE:
-				default: {
-					CRAWLEE_PURGE_ON_START = '1';
-				}
+			// Crawlee does auto purging
+			if (projectTypes.includes(PROJECT_TYPES.CRAWLEE)) {
+				CRAWLEE_PURGE_ON_START = '1';
 			}
-
-			if (language === LANGUAGE.PYTHON) {
+			// Apify SDK that does not have automatic purging (thats in crawlee)
+			// Scrapy standalone does not have automatic purging
+			// Unknown projects are assumed to be anything
+			else if (
+				projectTypes.includes(PROJECT_TYPES.APIFY_SDK) ||
+				projectTypes.includes(PROJECT_TYPES.SCRAPY) ||
+				projectTypes.includes(PROJECT_TYPES.UNKNOWN)
+			) {
 				await Promise.all([purgeDefaultQueue(), purgeDefaultKeyValueStore(), purgeDefaultDataset()]);
 				info({ message: 'All default local stores were purged.' });
 			}
-		}
-
-		// TODO: deprecate these flags
-		if (this.flags.purgeQueue && !this.flags.purge) {
-			await purgeDefaultQueue();
-			info({ message: 'Default local request queue was purged.' });
-		}
-
-		if (this.flags.purgeDataset && !this.flags.purge) {
-			await purgeDefaultDataset();
-			info({ message: 'Default local dataset was purged.' });
-		}
-
-		if (this.flags.purgeKeyValueStore && !this.flags.purge) {
-			await purgeDefaultKeyValueStore();
-			info({ message: 'Default local key-value store was purged.' });
-		}
```

Comment:
let's hope this won't be missed by anyone 🙃 worst case we can add it back 

btw I hope oclif wont silently ignore undeclared flags?

Reply:
that's good, if they are not doing anything we need to tell potential users